class.jetpack-connection-banner.php

```diff
@@ -25,6 +25,39 @@ private function __construct() {
 	}
 
 	/**
+	 * Checks whether the connection banner A/B test should be ran.
+	 *
+	 * @since 4.4.0
+	 *
+	 * @param null $now
+	 *
+	 * @return bool
+	 */
+	static function check_ab_test_allowed( $now = null ) {
```

Comment:
maybe rename to `ab_test_expired` for clarity? Or are we leaving this open to put different logic in that will disallow the AB test?